packages/zip-it-and-ship-it/src/runtimes/node/utils/included_files.ts

```diff
@@ -54,12 +54,11 @@ export const getPathsOfIncludedFiles = async (
     dot: true,
     ignore: excludePatterns,
     onlyFiles: false,
-    // get directories as well to get symlinked directories,
-    // to filter the regular non symlinked directories out mark them with a slash at the end to filter them out.
-    markDirectories: true,
     followSymbolicLinks: false,
+    expandDirectories: false,
   })
 
+  // tinyglobby ends directories with `/`, so we can filter them out using that
```

Comment:
fyi this is not guaranteed and the opposite can be true in rare (but known) cases. probably best to not rely on trailing /